### ISSUE TITLE
Dataset Portioning

### ISSUE THREAD

**Issue by aseprohman** [ultralytics/yolov5#5405]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello @glenn-jocher et all, 

what is the ideal portion ratio for the tain:validation set of the entire dataset that we have?

**Comment by glenn-jocher** [comment]
@aseprohman there's no hard rules on partitioning. Generally a 70/30 train/val split may be used, or a 70/20/10 val/train/test split if you have a test set. In our `autosplit()` function we apply a 90/10/0 split to better use the available train data:
https://github.com/ultralytics/yolov5/blob/7f9bbf0268317ace43f59174efbfecff60023c84/utils/datasets.py#L860-L882
